doc/query.md

```diff
@@ -119,6 +172,37 @@ query.put("age", gt12);
 
 See below for supported operators (Selections -> Conditions).
 
+#### Text search
+
+A text clause may be used (if a text index exists) as part of a query to perform full text search term matching, phrase matching, prefix wildcard matching, etc.  A text clause can stand on its own as a query or can be part of a compound query (see below).  Text search supports SQLite Enhanced Query Syntax, see [SQLite full text query][ftsQuery] for more details on syntax that is possible with text search.
+
+##### Restrictions
+
+- Only one text clause per query is permitted.
+- All clauses in a query must be satisfied by an index if that query contains a text search clause.
+
+To find all people named `mike` who `likes to read books` use:
+
```

Comment:
This section has also been re-written.  It should be clearer now with better examples.  Done in a24a9212694f953235bd45f01096d3f3b8a27053